tools/picard.wdl

```diff
@@ -866,3 +866,182 @@ task bam_to_fastq {
         maxRetries: 1
     }
 }
+
+task merge_vcfs {
+    meta {
+        description: "Merges the input VCF files into a single VCF file"
+        external_help: "https://gatk.broadinstitute.org/hc/en-us/articles/360036713331-MergeVcfs-Picard"
+        outputs: {
+            output_vcf: "The merged VCF file",
+            output_vcf_index: "The index file associated with the merged VCF file"
+        }
+    }
+
+    parameter_meta {
+        vcfs: "Input VCF format files to merge. May be gzipped or binary compressed."
+        vcfs_indexes: "Index files associated with the input VCF files"
+        output_vcf_name: "Name for the merged VCF file"
+        modify_disk_size_gb: "Add to or subtract from dynamic disk space allocation. Default disk size is determined by the size of the inputs. Specified in GB."
+    }
+
+    input {
+        Array[File] vcfs
+        Array[File] vcfs_indexes
+        String output_vcf_name
+        Int modify_disk_size_gb = 0
+    }
+
+    Int disk_size_gb = ceil(size(vcfs, "GiB") * 2) + 10 + modify_disk_size_gb
+
+    command <<<
+        picard -Xms2000m \
```

Comment:
It probably doesn't practically matter, but shouldn't this be linked to the `memory: "3 GB"` line?

Side note, why 3gb? I'm fairly sure that all our "low-mem" tasks use 4gb. 4 is a magic number in those cases, so 3 is fine here. We may just want to be consistent.

Reply:
I bumped it to 4 for consistency. `Xms` sets the initial heap size.